pkgs/development/libraries/qtstyleplugins/default.nix

```diff
@@ -10,7 +10,10 @@ mkDerivation {
     sha256 = "085wyn85nrmzr8nv5zv7fi2kqf8rp1gnd30h72s30j55xvhmxvmy";
   };
 
-  patches = [ ./fix-build-against-Qt-5.15.patch ];
+  patches = [
+    ./fix-build-against-Qt-5.15.patch
+    ./fix-gtk2-background.patch
```

Comment:
Please use fetchpatch with ``https://aur.archlinux.org/cgit/aur.git/plain/0002-fix-gtk2-background.patch?h=qt5-styleplugins`` instead.

Reply:
Fetch patch doesen't seem to work, but fetchurl does.

Reply:
Also i don't know, i think this should be consistent, either should provide both patches in the nixpkgs, or download both from the aur.
And i'm not really a fan of relying on the aur package link ;/
What do you think?

Reply:
Wer can't use fetchurl because it does not normalise the patch.

After the download is cached AUR can delete it and we still have it available.

I don't care if both patches are fetched or not but vendoring patches just makes the repo big and they need to be maintained or adapted.

Reply:
I found out why fetchpatch didn't work - it doesen't prompt for new hash if it is an empty string, but fetchurl does - i will try to see if i can fix it, otherwise i'll file a bug report.